<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dia_sim_config</class>
 <widget class="QDialog" name="dia_sim_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>751</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bDefault">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="GeneralTab">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Simulation Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leSimName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Working directory</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="leWorkingDir"/>
             </item>
             <item>
              <widget class="QPushButton" name="bSelectWorkingDir">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Solver</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbSolver">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ProblemTab">
         <attribute name="title">
          <string>Problem</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Initial Time</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leInitialTime">
            <property name="text">
             <string/>
            </property>
            <property name="unit" stdset="0">
             <string>&quot;s&quot;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Final Time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leFinalTime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Time Step</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leTimeStep"/>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="cbMaxIterations">
            <property name="text">
             <string>Max Iterations</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leMaxIterations"/>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="cbTol">
            <property name="text">
             <string>Tolerence</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leTol"/>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cbDerTol">
            <property name="text">
             <string>Der. Tolerance</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="leDerTol">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QGroupBox" name="cbOutput">
            <property name="title">
             <string>Output</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="3">
              <widget class="QCheckBox" name="checkBox_11">
               <property name="text">
                <string>cpu time</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>residual</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_9">
               <property name="text">
                <string>matrix condition number</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBox_7">
               <property name="text">
                <string>counter</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>solution</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>messages</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2" colspan="2">
              <widget class="QCheckBox" name="checkBox_10">
               <property name="text">
                <string>solver condition number</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>jacobian matrix</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="checkBox_8">
               <property name="text">
                <string>bailout</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>iterations</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QGroupBox" name="cbMethod">
            <property name="title">
             <string>Method</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_7">
             <item row="0" column="0">
              <widget class="QComboBox" name="method">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>--Select--</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Crank Nicolson</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ms</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Hope</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Third Order</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>bdf</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Implicit Euler</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QStackedWidget" name="methods_stack">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="select"/>
               <widget class="QWidget" name="cn"/>
               <widget class="QWidget" name="ms">
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Dif. radius:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="ms_differential_radius"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Alg. radius:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="ms_algebraic_radius"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="hope">
                <layout class="QFormLayout" name="formLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Dif. radius:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="hope_differential_radius"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Alg. radius:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="hope_algebraic_radius"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="thirdOrder">
                <layout class="QFormLayout" name="formLayout_5">
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Alg. radius:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="thirdOrder_algebraic_radius"/>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QCheckBox" name="cbAdHoc">
                   <property name="text">
                    <string>ad hoc</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="bdf">
                <layout class="QFormLayout" name="formLayout_6">
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Order</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="bdf_order"/>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>191</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>134</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="ie"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QGroupBox" name="cbPbCustomParam">
            <property name="title">
             <string>Custom Parameters</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPlainTextEdit" name="ptePbCustomParam"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ControlDataTab">
         <attribute name="title">
          <string>Control Data</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="cbCtrlCustomParam">
            <property name="title">
             <string>Custom Parameters</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPlainTextEdit" name="pteCtrlCustomParam"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>method</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>methods_stack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMaxIterations</sender>
   <signal>clicked(bool)</signal>
   <receiver>leMaxIterations</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTol</sender>
   <signal>clicked(bool)</signal>
   <receiver>leTol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDerTol</sender>
   <signal>clicked(bool)</signal>
   <receiver>leDerTol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMethod</sender>
   <signal>clicked(bool)</signal>
   <receiver>method</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dia_sim_config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dia_sim_config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDefault</sender>
   <signal>clicked()</signal>
   <receiver>dia_sim_config</receiver>
   <slot>setDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMethod</sender>
   <signal>clicked(bool)</signal>
   <receiver>methods_stack</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAdHoc</sender>
   <signal>clicked(bool)</signal>
   <receiver>thirdOrder_algebraic_radius</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSelectWorkingDir</sender>
   <signal>clicked()</signal>
   <receiver>dia_sim_config</receiver>
   <slot>SetWorkingDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setDefault()</slot>
  <slot>SetWorkingDirectory()</slot>
 </slots>
</ui>
